shared/components/ember-output.jsx

```diff
@@ -0,0 +1,90 @@
+/* global React */
```

Comment:
Does this file pass lint? If not, we might need to add `Ember` as a global up here.